plugins/modules/ec2_scaling_policy.py

```diff
@@ -267,18 +433,19 @@ def create_scaling_policy(connection, module):
         # it's only required if policy is SimpleScaling and state is present
         if not module.params['scaling_adjustment']:
             module.fail_json(msg='scaling_adjustment is required when policy_type is SimpleScaling '
-                             'and state is present')
+                                 'and state is present')
         params['ScalingAdjustment'] = module.params['scaling_adjustment']
         if module.params['cooldown']:
             params['Cooldown'] = module.params['cooldown']
 
-    if policy_type == 'StepScaling':
+    elif policy_type == 'StepScaling':
         if not module.params['step_adjustments']:
-            module.fail_json(msg='step_adjustments is required when policy_type is StepScaling '
-                             'and state is present')
+            module.fail_json(msg='step_adjustments is required when policy_type is StepScaling'
+                          'and state is present')
```

Comment:
it looks like your editor has "format on save" enabled, this creates a lot of unnecessary diff noise which can be difficult to review (now, and if we need to go through git history later for bugfixes).  would you be able to toggle this setting off for this repo and back out the formatting changes in this file please?